### ISSUE TITLE
Greymist Coastrunner Loot Pool

### ISSUE THREAD

**Issue by Soulex23** [dalaranwow/dalaran-wow#5983]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Greymist Coastrunners (lvl 12 mob in Darkshore around the first dead creature on shore, just south of Auberdine) are dropping Spellbinder Belt and Hunting Belt at an abnormally high rate, not quite but seemingly almost guaranteed. I killed 3 in a row and got both to drop every single time, a 4th dropped only the Hunting Belt, 5th was clean, 6th dropped both again.
**Expected behaviour**: Tell us what should happen instead
Spellbinder Belt and Hunting belt seem like they should be a much lower chance of dropping, when they should both be a 5.7% drop rate.
**Steps to reproduce the problem**:

1. Head to the first creature carcass in the Washed Ashore questline just south of Auberdine
2. Kill some of the Greymist Coastrunners 
3. Loot them, most drops are Spellbinder Belt and/or Hunting belt in addition to normal drops (coins/clams/other armor/etc.)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
https://wotlkdb.com/?npc=2202 Here it says that the drop rate for Spellbinder Belt and Hunter Belt are both 5.7%.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
(couldn't find the specific IDs but I do have Database pages for each):
https://wotlkdb.com/?npc=2202 (Greymist Coastrunner)
https://wotlkdb.com/?item=4690 (Hunting Belt)
https://wotlkdb.com/?item=4684 (Spellbinder Belt)
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![image](https://user-images.githubusercontent.com/92193061/149882740-5056ed98-0061-4657-906d-1401c4ed5ac7.png)
![image](https://user-images.githubusercontent.com/92193061/149882840-cd308d4f-1f57-417a-a3f6-0c803afe504f.png)
